lib/std/c/freebsd.zig

```diff
@@ -2172,6 +2172,56 @@ pub const PROC = struct {
     pub const WX_MAPPINGS_PERMIT = 0x0001;
     pub const WX_MAPPINGS_DISALLOW_EXEC = 0x0002;
     pub const WX_MAPPINGS_ENFORCE = 0x80000000;
+    pub const PROCCTL_MD_MIN = 0x10000000;
+    // specific x86_64 constants
+    pub const KPTI_CTL = switch (builtin.cpu.arch) {
```

Comment:
That are alot of switch else branches cluttering the code. It looks better to me to namespace `KPTI` and `LA`:
```zig
pub const KPTI = switch (builtin.cpu.arch) {
    .x86_64 => struct {
        pub const CTL = PROCCTL_MD_MIN,
        // ..
    },
    else => void,
};
```

FreeBSD supports [many platforms](https://www.freebsd.org/platforms/). At which points do you think should we separate out the platform-specific bits?

Reply:
thanks for the tip, for few years there only had been those few x86 64 constants, nothing for arm or anything else (yet).